plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/writer/TransactionLogWriter.java

```diff
@@ -107,7 +107,7 @@ public void flush()
         }
 
         String clusterId = commitInfoEntry.get().getCommitInfo().getClusterId();
-        logSynchronizer.write(session, clusterId, logEntry, bos.toByteArray());
+        logSynchronizer.write(session, clusterId, new Path(logEntry), bos.toByteArray());
```

Comment:
Can we migrate logSynchronizer as well? can be follow up, I know there are a lot of these

Reply:
I was planning to do that in separate commit as some implementation of `LogSynchronizer` are strongly coupled with `org.apache.hadoop.fs.Path `